src/EditorFeatures/CSharp/AutomaticCompletion/Sessions/CurlyBraceCompletionSession.cs

```diff
@@ -114,9 +110,9 @@ private IEnumerable<IFormattingRule> GetFormattingRules(Document document)
             return SpecializedCollections.SingletonEnumerable(BraceCompletionFormattingRule.Instance).Concat(Formatter.GetDefaultFormattingRules(document));
         }
 
-        private void FormatTrackingSpan(IBraceCompletionSession session, IEnumerable<IFormattingRule> rules = null)
+        private void FormatTrackingSpan(IBraceCompletionSession session, bool isAfterSessionStart, IEnumerable<IFormattingRule> rules = null)
```

Comment:
Is there a better name than this?  This encodes who the callers are, not how it changes the behavior of this method.